src/client.ts

```diff
@@ -30,7 +30,11 @@ const HTTP_DATA_PATH = 'https://data.spiceai.io';
 const FLIGHT_PATH = 'flight.spiceai.io:443';
 
 const PROTO_PATH = './proto/Flight.proto';
-const fullProtoPath = path.join(__dirname, PROTO_PATH);
+// If we're running in a Next.js environment, we need to adjust the path to the proto file
+const PACKAGE_PATH = __dirname.includes('/.next')
+  ? __dirname.replace(/\/\.next\/.*$/, "/node_modules/@spiceai/spice/dist")
```

Comment:
Should the replacement be more targeted? E.g. `.next/aaaa/bbbb/` ->

Reply:
`.next` is located next to `node_modules`, so if current __dirname has it, then it is already a wrong path to proto file

Reply:
but actually makes sense to change to `.next/app/server`